.github/workflows/release.yml

```
@@ -0,0 +1,129 @@
+          script: |
+            const tag = core.getInput('tag-name');
+            const ref = `refs/tags/${tag}`;
+            const sha = context.sha;
+            await github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: ref,
+              sha: sha
+            });
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          tag-name: ${{ github.event.inputs.tag-name }}
```

actionlint (1.7.7)

input "tag-name" is not defined in action "actions/github-script@v7". available inputs are "base-url", "debug", "github-token", "previews", "result-encoding", "retries", "retry-exempt-status-codes", "script", "user-agent"

(action)